lighthouse-core/test/report/v2/report-generator-test.js

```diff
@@ -143,6 +146,20 @@ describe('ReportGeneratorV2', () => {
       assert.ok(result.includes('LIGHTHOUSE_JAVASCRIPT'), 'cannot be tricked');
     });
 
+    it('should inject the report templates', () => {
+      const page = jsdom.jsdom(new ReportGeneratorV2().generateReportHtml({}));
+      const templates = jsdom.jsdom(TEMPLATES_FILE);
+
+      assert.deepStrictEqual(page.querySelectorAll('template[id^="tmpl-"]'),
```

Comment:
just checked, `assert.deepStrictEqual` also doesn't work here (besides checking that there's the same number of templates) because `Object.keys()` on the two `HTMLTemplateElement`s returns `[]`, so any two `HTMLTemplateElement`s are considered deep equal. You'll probably need to manually check the innerHTMLs are equal or something